CHANGELOG.md

```diff
@@ -31,7 +31,35 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 # Changelog
 
-All notable changes to this project will be documented in this file.
+## [Unreleased](https://github.com/babylonchain/babylon-private/tree/HEAD)
+
+[Full Changelog](https://github.com/babylonchain/babylon-private/compare/euphrates-0.2.0-rc.0...HEAD)
+
+
+## [euphrates-v0.2.0-rc.0](https://github.com/babylonchain/babylon-private/tree/euphrates-v0.2.0-rc.0) (2024-05-17)
+
+[Full Changelog](https://github.com/babylonchain/babylon-private/compare/euphrates-0.1.0-rc.1...euphrates-v0.2.0-rc.0)
+
+## [euphrates-0.1.0-rc.1](https://github.com/babylonchain/babylon-private/tree/euphrates-0.1.0-rc.1) (2024-03-25)
```

Comment:
tbh I would delete all those sub points and in `## Unreleased` section add `Babylon Multi staking support`
pointing to this pr. (probably under State breaking sub titlte`)

And in this pr description add links to most relevat pr's